packages/expo-image/src/ExpoImage.web.tsx

```diff
@@ -188,6 +190,11 @@ export default function ExpoImage({
 
   const resolvedSources = ensureIsArray(source).map(resolveAssetSource);
 
+  const fetchPriority =
+    { [ImagePriority.HIGH]: 'high', [ImagePriority.LOW]: 'low', [ImagePriority.NORMAL]: 'auto' }[
+      priority || ImagePriority.NORMAL
+    ] ?? 'auto';
```

Comment:
Formatting here doesn't look good 😅 Maybe it's worth separating to another function?